### PR TITLE
Draft for Nature Index chapter

### PR DESCRIPTION
**Pull request by ChloeRN**
I have written a first draft for the Nature Index chapter. 
All code is function-based and using wrappers / helper functions I wrote in a new package NIflex (https://github.com/NINAnor/NIflex), so code review may have to include the package's repository as well. 

At the moment, the code needs NI database user credentials to be specified as environmental variables, so you will need to set this up individually to make it work. Siw and I are working on setting up a "dummy user" for the database, for which we can then add credentials as project environmental variables. 

On request of @HannoSandvik I am submitting this already now even though it is not completely finished so that code review can start.
I have a handful of additions to make before the chapter can be considered complete, but I will not have time to deal with these until September. They are: 

- [x] Add table with overview of indicators that are part of NI Våtmark
- [ ] Add results of discussion on NI inclusion in ØT (discussion will involve me, @anders-kolstad, and @HannoSandvik)
- [x] Add references
- [x] Add function to make georeferenced end-product (will involve code already written for `NIflex::plotNI_Map()`

### CODE REVIEW

**Comment by ChloeRN** [comment]
Included indicator overview in 45c01d2a042ba12160823b6e367d50520d0ba9f0

**Comment by ChloeRN** [comment]
Final adjustments complete. 
The only thing missing is reporting the results of the discussion with @anders-kolstad and @HannoSandvik  (scheduled for October) and to update NIflex code accordingly.

File: .Renviron [hunk 1]
```diff
@@ -0,0 +1,2 @@
+NIDB_USRNAME="lesebruker@nina.no"
```
Comment (DrMattG) [inline .Renviron:1]: Check that it is okay that these are publically visable

File: DRAFT_nature_index.Rmd [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+                              norwegianNames = norwegianNames,
+                              saveSteps = FALSE)
+```
+
+After calculating the index values, we can extract summary metrics for a 
+specified year and match them to geospatial data for further analysis and/or 
+inclusion in the Ecosystem Condition framework: 
+
+```{r}
+CustomNI_map <- geomapNI(Index = CustomNI$CustomIndex,
```
Comment (DrMattG) [inline DRAFT_nature_index.Rmd:280]: I get stuck here due to missing permissions on the folder (awaiting update from datahjelp) 
Quitting from lines 283-288 [unnamed-chunk-17] (DRAFT-Nature-Index-product.Rmd)
Error:
! Cannot open "/data/P-Prosjekter/41201611_naturindeks_2021_2023_vitenskapelig_ansvar/Shapefiles/NI_ecosystems/NI_ecosystem_Terrestrial"; The file doesn't seem to exist.

**Review by DrMattG — CHANGES_REQUESTED** [review]

**Comment by DrMattG** [comment]
I will close this here as I have rendered this chapter to the book using my own branch (should be available in 12 hours) and @HannoSandvik can review it next week.

**Pull request closed without merging**